frontend/src/components/DragNDroppableComponent.vue

```diff
@@ -72,18 +72,11 @@ export default {
 </script>
 
 <style lang="scss" scoped>
-
   .full-height {
```

Comment:
May use fill-height and 100% width could be the default of a div

Reply:
using fill-height, but have to set the width to 100%